README.md

```diff
@@ -2,10 +2,63 @@
 # GRIP Query UI
 
 
-## Setup 
-To connect server against BMEG.io, get `bmeg_credentials.json` file from [BMEG](https://bmeg.io/analyze/access) 
+# Example server
+
+## Setting up Grip
+
+```
+grip server
```

Comment:
Returns `Error: File not found: server`

Reply:
This likely is the reason downstream steps return errors in sections `Load example graph` `Setting up example GDC graph` `Turning on GripQuery web UI`

Reply:
What does the command `grip` return?